src/less/variables.less

```diff
@@ -175,10 +181,12 @@
 
 // Buttons
 //-------------------------
-@btn-default-color:           #fff;
-@btn-default-bg:              @gray-400;
+@btn-default-color:           @black;
+@btn-default-bg:              @body-bg;
+@btn-default-border:          @gray-150;
+
 
-@btn-primary-color:           @btn-default-color;
+@btn-primary-color:           @body-bg;
```

Comment:
Why was btn-default-color removed? This shouldn't be body-bg as body-bg will be an off white color.